inference_experimental/inference_exp/models/rfdetr/rfdetr_backbone_pytorch.py

```diff
@@ -89,18 +94,33 @@ def __init__(
 
             dino_config["return_dict"] = False
             dino_config["out_features"] = [f"stage{i}" for i in out_feature_indexes]
+            implied_resolution = positional_encoding_size * patch_size
+
+            if implied_resolution != dino_config["image_size"]:
+                print(
```

Comment:
we should think about how we want to do logging long term.  Thinking both for general dev-ex and potentially structured logging or log files for analytics / debugging purposes.